apps/web/src/app/shared/services/JSON-LD-GUIDE.md

```
@@ -0,0 +1,264 @@
+export class HomePage implements OnInit {
+  private readonly seoService = inject(SeoService);
+
+  ngOnInit(): void {
+    // Automaticamente cria meta tags + JSON-LD do tipo WebSite
+    this.seoService.setHomeSeo();
+  }
+}
+```
+
+**JSON-LD gerado:**
+```json
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)